legal-api/src/legal_api/core/filing.py

```diff
@@ -452,7 +452,11 @@ def ledger(business_id: int,  # pylint: disable=too-many-arguments
     def common_ledger_items(business_identifier: str, filing_storage: FilingStorage) -> dict:
         """Return attributes and links that also get included in T-business filings."""
         no_output_filing_types = ['Involuntary Dissolution', 'conversion']
+
+        # This is a temporary fix so that filingLink, commentLink, and documentLink won't break
+        # Revert to the original base_url below once the Migration to GCP is complete
```

Comment:
No, not revert. Change to using API GW URL.

Reply:
Will update

Reply:
Perfect, thanks!